netty/build.gradle

```diff
@@ -21,13 +21,15 @@ dependencies {
     implementation libraries.netty_proxy_handler,
             libraries.guava,
             libraries.errorprone,
-            libraries.perfmark
+            libraries.perfmark,
+            libraries.netty_unix_common
 
     // Tests depend on base class defined by core module.
     testImplementation project(':grpc-core').sourceSets.test.output,
             project(':grpc-api').sourceSets.test.output,
             project(':grpc-testing'),
-            project(':grpc-testing-proto')
+            project(':grpc-testing-proto'),
+            ("io.netty:netty-transport-native-epoll:${nettyVersion}")
```

Comment:
This one also needs to reference `libraries`. Note that we probably don't care and could just use libraries.netty_epoll instead of the more precise dep string here.

Reply:
Made the change. I thought one of the earlier comments suggested using the more specific one.